### PR TITLE
New feature

### PR DESCRIPTION
**Pull request by lmargal**
User story finished

### CODE REVIEW

**Comment by lmargal** [comment]
a comment

**Review by lmargal — COMMENTED** [review]
asdf

File: sfd [hunk 1]
```diff
@@ -1,2 +1,2 @@
 sfsd
```
Comment (lmargal) [inline sfd:1]: rgsgdg

File: sfd [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
-sdfsd
+sd
```
Comment (lmargal) [inline sfd:2]: line comment

**Review by lmargal — COMMENTED** [review]
sdefgsd

**Comment by lmargal** [comment]
a normal comment

**Comment by lmargal** [comment]
comment

**Review by lmargal — COMMENTED** [review]
review comment

**Review by lmargal — COMMENTED** [review]
review comment

**Review by lmargal06 — APPROVED** [review]
review approve

**Review by lmargal06 — CHANGES_REQUESTED** [review]
request changes

**Pull request closed without merging**